server/util.go

```diff
@@ -280,13 +286,79 @@ func dumpBinaryRow(buffer []byte, columns []*ColumnInfo, row chunk.Row) ([]byte,
 	return buffer, nil
 }
 
-func dumpTextRow(buffer []byte, columns []*ColumnInfo, row chunk.Row) ([]byte, error) {
+type textDumper struct {
```

Comment:
Please add some UT for textDumper‘s functions.

Reply:
Done.